src/facade/reply_builder.h

```diff
@@ -150,8 +162,17 @@ class RedisReplyBuilder : public SinkReplyBuilder {
   static std::string_view StatusToMsg(OpStatus status);
 
  private:
+  enum Resp3Type {
```

Comment:
I would rename this to collection type.
You use it in SendStringCollection which gets some a collection type and only if resp3 is enabled it does some changes based on the type but the type of the collection is not relevant if resp3 is enabled or not